org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/refactoring/IResourceRelocationStrategy.xtend

```diff
@@ -29,6 +29,29 @@ interface IResourceRelocationStrategy {
 
 	def void applyChange(ResourceRelocationContext context)
 
+	/**
+	 * By overriding this method client implementations may explicitly demand for relying
+	 * on the persisted index during a refactoring. 
+	 * 
```

Comment:
if you use paragraphs, please wrap the paragraphs except the first one in `<p>...</p>`

Reply:
Its not my intention to use paragraphs, just introduced some empty lines to make the text easier readable.

Reply:
and you would expect them to be seen in javadoc as well. so they are paragraphs.